lib/nuget-parser/index.ts

```diff
@@ -110,14 +121,21 @@ export async function buildDepGraphFromFiles(
     }
   }
 
-  // TODO: OSM-347 - use more than just the first one we find
-  const targetFramework = targetFrameworks[0];
+  // TODO: OSM-347 - return vulnerability scans for all
+  let decidedTargetFramework: string;
+  if (!targetFramework) {
+    console.log(`No targetFramework supplied, defaulting to using the first in the list of the manifest (\x1b[1m${targetFrameworks[0]}\x1b[0m).
+Supply a targetFramework by using the \x1b[1m--target-framework\x1b[0m argument.`);
+    decidedTargetFramework = targetFrameworks[0];
```

Comment:
Plan is to refactor this function into an iterator and feed it each target framework separately, thus removing this block altogether.